card/src/Card.js

```diff
@@ -2,28 +2,29 @@ import React, { Component } from "react";
 import style from "./Card.module.css";
 import Image1 from "./nature.jpg";
 import Button from "./Button.js";
+import Cardcomp from "./Cardcomp";
 class Card extends Component {
   constructor(props) {
     super(props);
-    this.state = { name: "a.m" };
+    this.state = { name: "a.m", users: [], isLoading: true };
   }
+  componentDidMount = async () => {
+    const result = await fetch("https://api.github.com/users");
```

Comment:
make use of axios instead of fetch and learn the difference between fetch and axios
first you need to install axios package to make use of it

Reply:
done